views/layer/vector.html

```diff
@@ -71,6 +71,16 @@
           </select>
         </div>
       </div>
+      <div class="form-row align-items-center pb-1" ng-if="vector.showAltitudeModes">
+        <div class="col-auto" title="Choose the layer's altitude mode.">
```

Comment:
Move the `title` up a level so it displays to the user when hovering the control as well as the label.

I would also suggest making the title a bit more descriptive. Something like:
`Choose how the layer interprets altitude in 3D mode.`

Reply:
Since the control doesn't appear in 2D mode should I still add "in 3D mode"?